app/src/test/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileViewModelTest.kt

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.pinnedtile
+
+import android.arch.lifecycle.Observer
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+/**
+ * Unit tests for the [PinnedTileViewModel].
+ */
+@RunWith(RobolectricTestRunner::class)
+class PinnedTileViewModelTest {
```

Comment:
This is more of an integration test between the repo and VM, which is fine, but if we mocked out the repo we could run this file without Robolectric (which would mean these tests would run _**much**_ more quickly).

Reply:
That said, there are advantages to integration tests as well.  This isn't something that necessarily needs to be changed, but we should be careful about using too much Robolectric, since it has a pretty dramatic effect on how long our tests take to run.